docs/website/content/v0.3/en/components/apid.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'apid'
+---
+
+When interacting with Talos, the gRPC api endpoint you will interact with directly is `apid`.
+Apid acts as the gateway for all component interactions.
+Apid provides a mechanism to route requests to the appropriate destination.
+
+We'll use some examples below to illustrate what `apid` is doing.
+
+When a user wants to interact with a Talos component via `osctl`, there are two flags that control the interaction with `apid`.
+The `-e | --endpoints` flag is used to denote which Talos node ( via `apid` ) should handle the connection.
+Typically this is a public facing server.
+The `-n | --nodes` flag is used to denote which Talos node(s) should respond to the request.
```

Comment:
I think we should say that if `--nodes` is not set, `osctl` talks directly to the endpoint without proxying. So default might be `--nodes = --endpoints[0]`?